File: tests/org/cappuccino_project/ide/intellij/plugin/parser/ObjJParserTest.java
package org.cappuccino_project.ide.intellij.plugin.parser;

import com.intellij.testFramework.ParsingTestCase;

public class ObjJParserTest  extends ParsingTestCase {
    public ObjJParserTest() {
        super("", "objj", new ObjJParserDefinition());
    }

    public void testParsingTestData() {
        doTest(true);
    }

    @Override
    protected String getTestDataPath() {
        return "code_samples/simple_language_plugin/testData";
    }

    @Override
    protected boolean skipSpaces() {
        return false;
    }

    @Override
    protected boolean includeRanges() {
        return true;
    }
}

File: tests/org/cappuccino_project/ide/intellij/plugin/parser/ObjJParserTest.kt
package org.cappuccino_project.ide.intellij.plugin.parser

import cappuccino.ide.intellij.plugin.parser.ObjJParserDefinition
import com.intellij.testFramework.ParsingTestCase

class ObjJParserTest : ParsingTestCase("", "objj", ObjJParserDefinition()) {

    fun testParsingTestData() {
        doTest(true)
    }

    @Override
    override fun skipSpaces(): Boolean {
        return false
    }

    @Override
    override fun includeRanges(): Boolean {
        return true
    }
}
